cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportUtil.java

```diff
@@ -137,7 +137,10 @@ public static void writeModelDescription(Network network, CgmesSubset subset, XM
         }
         writer.writeStartElement(MD_NAMESPACE, "FullModel");
         writer.writeAttribute(RDF_NAMESPACE, CgmesNames.ABOUT, modelDescription.getId());
-        context.getReportNode().newReportNode().withMessageTemplate("CgmesId", modelDescription.getId()).add();
+        context.getReportNode().newReportNode()
+                .withMessageTemplate("CgmesId", "${cgmesid}")
```

Comment:
Agree on using a more specific key and setting its value as a constant and also on setting as a constant the name of the variable in the message. Both will be used outside to recover the list of exported model identifiers.

We can not move the new report node creation to `initializeModelId` because we want to report all the identifiers of exported models, not only the ones that have been created during export. I will propose to name the key as `ExportedCgmesId`.